leaderboards/grade_leaderboard.py

```diff
@@ -27,7 +27,8 @@ def normalize_id(key):
 @XBlock.needs("i18n")
 class GradeLeaderboardXBlock(LeaderboardXBlock):
     STUDENT_VIEW_TEMPLATE = "grade_leaderboard.html"
-    GRADE_SOURCES = (EdxLmsGradeSource, MockGradeSource)  # Ordered list of class types that know how to get student grades
+    # Ordered list of class types that know how to get student grades
+    GRADE_SOURCES = (EdxLmsGradeSource, MockGradeSource)
```

Comment:
@e-kolpakov No, it is also used in the workbench, where we can't inject it. The EdxLmsGradeSource is currently using the `try import ... except ImportError`.

But since it does make sense to stop MockRuntime from being used as a fallback in normal circumstances, I have pushed a new commit that restricts it to the workbench and test runtimes. Let me know if looks good to you.

(Also FYI the grades_source part of the code is likely going to be rewritten next sprint.)